app/marketing/mails.py

```diff
@@ -747,7 +787,24 @@ def new_bounty_daily(bounties, old_bounties, to_emails=None):
     plural = "s" if len(bounties) != 1 else ""
     worth = round(sum([bounty.value_in_usdt for bounty in bounties if bounty.value_in_usdt]), 2)
     worth = f" worth ${worth}" if worth else ""
-    subject = _(f"⚡️  {len(bounties)} New Open Funded Issue{plural}{worth} matching your profile")
+    offers = f""
+    if to_emails and len(to_emails):
+        from townsquare.utils import is_email_townsquare_enabled, is_there_an_action_available
```

Comment:
is this to avoid a circular import? Performance isn't a huge deal here since its outside of a request/response cycle but always good to check

Reply:
nah; just laziness.. removing